CMakeLists.txt

```diff
@@ -54,13 +54,12 @@ if(MSVC)
 
   if(DEFINED ROOT_SOURCE_DIR)
     # ROOT_BINDIR is set by ROOTConfig.cmake
-    get_filename_component(realfp "${ROOT_BINDIR}/../" ABSOLUTE)
-    file(TO_CMAKE_PATH ${realfp} ROOTSYS)
-    # FIXME: Get from root-config.bat --tutdir
-    set(ROOT_TUTORIALS_DIR ${ROOTSYS}/tutorials)
-    #set(ROOT_CONFIG_EXECUTABLE root-config.bat)
-    #exec_program(${ROOT_CONFIG_EXECUTABLE} ARGS "--tutdir" OUTPUT_VARIABLE ROOT_TUTORIALS_DIR)
-    #file(TO_CMAKE_PATH ${ROOT_TUTORIALS_DIR} ROOT_TUTORIALS_DIR)
+    set(ROOT_CONFIG_EXECUTABLE ${ROOT_BINDIR}/root-config.bat)
+    exec_program(${ROOT_CONFIG_EXECUTABLE} ARGS "--prefix" OUTPUT_VARIABLE ROOTSYS RETURN_VALUE RETVAR)
+    file(TO_CMAKE_PATH ${ROOTSYS} ROOTSYS)
```

Comment:
This pattern is repeated multiple time.  Consider extracting it to a function.

Reply:
I don't think it's worth it...